pkg/nsx/services/vpc/builder.go

```diff
@@ -76,7 +76,11 @@ func buildNSXVPC(obj *v1alpha1.NetworkInfo, nsObj *v1.Namespace, nc common.VPCNe
 			},
 		}
 		vpc.SiteInfos = siteInfos
-		vpc.LoadBalancerVpcEndpoint = &model.LoadBalancerVPCEndpoint{Enabled: &DefaultLoadBalancerVPCEndpointEnabled}
+		loadBalancerVPCEndpointEnabled := false
+		if useAVILB {
+			loadBalancerVPCEndpointEnabled = true
+		}
+		vpc.LoadBalancerVpcEndpoint = &model.LoadBalancerVPCEndpoint{Enabled: &loadBalancerVPCEndpointEnabled}
```

Comment:
Can we only set LoadBalancerVpcEndpoint when useAVILB is true?

```
		if useAVILB {
			vpc.LoadBalancerVpcEndpoint = &model.LoadBalancerVPCEndpoint{Enabled: &loadBalancerVPCEndpointEnabled}
		}
```

Reply:
Sure. Fixed.